deploy/README.md

```diff
@@ -45,122 +38,20 @@ terraform apply # rerun after any change to main.tf
-
-```bash
-sudo htpasswd -B /etc/nginx/.htpasswd new-username
-```
-
-Configure terracotta to run under gunicorn:
-
-```bash
-sudo chown :www-data /var/www/tileserver/raster/terracotta.sock
-sudo -u www-data curl --unix-socket /var/www/tileserver/raster/terracotta.sock http
-```
-
-Transfer tileserver raster data to the server, then ingest to terracotta:
-
-```bash
-terracotta ingest \
-    "/var/www/tileserver/raster/data/{type}__rp_{rp}__rcp_{rcp}__epoch_{epoch}__conf_{confidence}.tif" \
-    -o "/var/www/tileserver/raster/terracotta.sqlite"
-```
-
-`config/` directory contains:
-
-- nginx config to serve frontend assets directly and proxy tile requests to the
-  tileserver
-- systemd service config to run the vector and raster tileservers as services
-
-`deploy.sh` builds the frontend for deployment, uploads the build directory,
-data and tileserver config to a server, and restarts the tileservers. It assumes
-that whoever runs the script has ssh/public key access to the server.
+install docker and docker compose.
 
-## Deploy update
+`deploy.sh` uploads data to the server. It assumes that whoever runs the script
+has ssh/public key access to the server.
 
-Usually this involves running `deploy.sh` and that will be sufficient.
-
-If you need to restart the raster tileserver:
-
-```bash
-# restart the service
-sudo systemctl restart terracotta
-# check nginx can access the socket
-sudo chown :www-data /var/www/tileserver/raster/terracotta.sock
+Update the .env file with the relevant database connection parameters:
 ```
-
-
-## Deploy backend
-
-Initial setup
-
-```bash
-# Python 3.10 on Ubuntu 20.04 needs to use PPA
-sudo add-apt-repository ppa:deadsnakes/ppa
-sudo apt install python3.10  python3.10-distutils
-curl https://bootstrap.pypa.io/get-pip.py | sudo python3.10
-# Install/upgrade Terracotta app requirements
-sudo python3.10 -m pip install --upgrade cython
-sudo python3.10 -m pip install --upgrade numpy
-sudo python3.10 -m pip install --upgrade --no-binary rasterio rasterio==1.3a4
-sudo python3.10 -m pip install --upgrade gunicorn terracotta[recommended]
-sudo python3.10 -m pip install testresources
-
-# Create backend working directory
-sudo mkdir /var/www/backend
-sudo chown -R :ubuntu /var/www/backend/
-sudo chmod -R 775 /var/www/backend/
-```
-
-Run `deploy.sh` locally to copy over the backend package, including Python
-package `requirements.txt`
-
-```bash
-# Install Python requirements
-cd /var/www/backend
-sudo python3.10 -m pip install -r requirements.txt
-
-# Create backend service
-sudo touch /etc/systemd/system/backend.service
-# edit or copy from ./config/etc/systemd/system/backend.service
-sudo systemctl start backend.service
-# Check status
-systemctl status backend
-journalctl -u backend
-```
-
-Copy secret `PG*` variables for connection to database in EnvironmentFile as
-configured in backend.service, something like:
-
-```conf
 PGDATABASE=jamaica
 PGUSER=docker
 PGPASSWORD=docker
 PGHOST=localhost
```

Comment:
Note MySQL details here too? - or point to the `backend/README.md` which has more info.